### DISCUSSION TITLE
posts/2022/06/using-icloud-mail-custom-domain/

### DISCUSSION THREAD

**Discussion by giscus** [brycewray/comments#38] [Announcements]
# posts/2022/06/using-icloud-mail-custom-domain/

It took Apple a while to get it (maybe) right — but, it’s “so far, so good” on something announced at last year’s WWDC.

https://www.brycewray.com/posts/2022/06/using-icloud-mail-custom-domain/

**Comment by Stardenver** [comment]
Long time iPhone and iPad user here and finally made the transition from Windows to MacBook which is about to arrive here soon. To be prepared, I am looking for information regarding iCloud and its features. As I am tired of running my own mail server, I am looking interesting in learning about iCloud mail with custom domains. So far I learned 2 things:

1. according to the low numbers of experience reports and posts it seems like nobody is actually using iCloud with custom domains. Its pretty hard to find something outside of reddit

2. many Apple/iCloud users seem to be using Fastmail and I am wondering why. Digging through a lot of topics on reddit I read about fastmail over and over again. Is it a thing for Apple users?

Using my own mailserver atm (actually two of them - mailcow and cloudron) and the reason for me to think about iCloud with custom domain is the fact, that I am already paying for it and - if I remember right - its the only way to get real push notifications within iOS Mail app. So I wouldn't have to pay any additional fee plus I could retire my own mailserver and save some bucks there as well.

Gonna read your following up now. Thanks for the information provided so far. Not a native speaker - so sorry for any confusion.

**Reply by brycewray** [reply]
@Stardenver Thanks, and good luck on your endeavors. I hope this post and its various follow-up posts (links below) will help you.

- https://www.brycewray.com/posts/2022/06/using-icloud-mail-custom-domain-following-up/
- https://www.brycewray.com/posts/2022/08/back-to-fastmail/
- https://www.brycewray.com/posts/2022/09/using-icloud-mail-custom-domain-return/
- https://www.brycewray.com/posts/2022/09/using-icloud-mail-custom-domain-retreat/
- https://www.brycewray.com/posts/2022/11/using-icloud-mail-custom-domain-its-fifty-bucks/
- https://www.brycewray.com/posts/2023/01/back-to-fastmail-redux/
